docker-compose.yml

```diff
@@ -56,6 +56,31 @@ services:
     depends_on:
       - mongo
 
+  mongo-pm:
```

Comment:
can you rename this database to mongo-history?

Reply:
Sure, solved with [432d209](https://github.com/pagopa/pagopa-ecommerce-helpdesk-service/pull/138/commits/432d209f7f5ec1aed32d06a336f6fdb80ee3ca68) and [5e67c1f](https://github.com/pagopa/pagopa-ecommerce-helpdesk-service/pull/138/commits/5e67c1f4839f086f29d372d172983616a4766beb)